apps/example-app/app.config.ts

```diff
@@ -0,0 +1,25 @@
+import { ExpoConfig, ConfigContext } from "@expo/config"
+
+/**
+ * Use ts-node here so we can use TypeScript for our Config Plugins
+ * and not have to compile them to JavaScript
+ */
+require("ts-node/register")
+
+/**
+ * @param config ExpoConfig coming from the static config app.json if it exists
+ * 
+ * You can read more about Expo's Configuration Resolution Rules here:
+ * https://docs.expo.dev/workflow/configuration/#configuration-resolution-rules
+ */
+module.exports = ({ config }: ConfigContext): Partial<ExpoConfig> => {
+  const existingPlugins = config.plugins ?? []
+
+  return {
+    ...config,
+    plugins: [
+      ...existingPlugins,
+      require("./plugins/withSplashScreen").withSplashScreen,
+    ],
+  }
+}
```

Comment:
I would remove this file.

Reply:
Removed https://github.com/infinitered/reactotron/pull/1411/commits/89ab58464296f32d1900f7aafeed1b9576284c9d